packages/medusa/src/api/routes/admin/sales-channels/create-sales-channel.ts

```diff
@@ -0,0 +1,51 @@
+import { Request, Response } from "express"
+import { IsObject, IsOptional, IsString } from "class-validator"
+
+import { validator } from "../../../../utils/validator"
+import SalesChannelService from "../../../../services/sales-channel"
+
+/**
+ * @oas [post] /sales-channels
+ * operationId: "PostSalesChannel"
+ * summary: "Create a SalesChannel"
+ * description: "Creates a SalesChannel."
+ * x-authenticated: true
+ * parameters:
+ *   - (body) name=* {string} Name of the sales channel
+ *   - (body) description=* {string} Description of the sales channel
+ *   - (body) metadata {object} Metadata for the channel.
```

Comment:
todo: rm, there is no metadata on the sales channels. But should we add a metadata field?